contracts/token/ERC721/extensions/ERC721Votes.sol

```diff
@@ -16,18 +16,16 @@ import {Votes} from "../../../governance/utils/Votes.sol";
  */
 abstract contract ERC721Votes is ERC721, Votes {
     /**
-     * @dev See {ERC721-_afterTokenTransfer}. Adjusts votes when tokens are transferred.
+     * @dev See {ERC721-_update}. Adjusts votes when tokens are transferred.
      *
      * Emits a {IVotes-DelegateVotesChanged} event.
      */
-    function _afterTokenTransfer(
-        address from,
-        address to,
-        uint256 firstTokenId,
-        uint256 batchSize
-    ) internal virtual override {
-        _transferVotingUnits(from, to, batchSize);
-        super._afterTokenTransfer(from, to, firstTokenId, batchSize);
+    function _update(address to, uint256 tokenId, address operatorCheck) internal virtual override returns (address) {
+        address from = super._update(to, tokenId, operatorCheck);
+
+        _transferVotingUnits(from, to, 1);
+
+        return from;
```

Comment:
Because both `_update` or `_transferVotingUnits` can't be reentered, I think we would prefer:

```suggestion
        _transferVotingUnits(from, to, 1);
        return super._update(to, tokenId, operatorCheck);
```

The rationale is that we've previously expressed that we don't like passing the `from` around from the beginning to the end because it may lead to unintentional overwrites.

Reply:
Ok no closing, this was an incorrect suggestion, the problem is that we clearly need `from` first.